aws/resource_aws_securityhub_product_subscription_test.go

```diff
@@ -0,0 +1,104 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/service/securityhub"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func testAccAWSSecurityHubProductSubscription_basic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckAWSSecurityHubAccountDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSSecurityHubProductSubscriptionConfig_basic,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAWSSecurityHubProductSubscriptionExists("aws_securityhub_product_subscription.example"),
+				),
+			},
+			// Import is not supported, since the API to lookup product_arn from a product
```

Comment:
```go
{
  ResourceName: "aws_securityhub_product_subscription.example",
  ImportState: true,
  ImportStateVerify: true,
}
```

Reply:
I've added an import test step, but I've need to use `ImportStateVerifyIgnore`:

```go
{
  ResourceName:            "aws_securityhub_product_subscription.example",
  ImportState:             true,
  ImportStateVerify:       true,
  ImportStateVerifyIgnore: []string{"product_arn"},
},
```